sources/packages/api/src/route-controllers/report/report.aest.controller.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Post,
+  Res,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+import {
+  ApiBadRequestResponse,
+  ApiTags,
+  ApiUnprocessableEntityResponse,
+} from "@nestjs/swagger";
+import { Response } from "express";
+import { Readable } from "stream";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { ReportService, FormService } from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
+import { StringBuilder } from "../../utilities/string-builder";
+import {
+  getDateOnlyFormatAsString,
+  getFileNameTimestamp,
+  CustomNamedError,
+} from "../../utilities";
+import BaseController from "../BaseController";
+import { ReportsFilterAPIInDTO } from "./models/report.dto";
+import { FormNames } from "../../services/form/constants";
+import {
+  REPORT_CONFIG_NOT_FOUND,
+  FILTER_PARAMS_MISMATCH,
+} from "../../services/report/constants";
+
+/**
+ * Controller for Reports for AEST Client.
+ * This consists of all Rest APIs for notes.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("reports")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-reports`)
+export class ReportAESTController extends BaseController {
+  constructor(
+    private readonly reportService: ReportService,
+    private readonly formService: FormService,
+  ) {
+    super();
+  }
+
+  /**
+   * Rest API to financial reports in csv format.
```

Comment:
Did you intend to write "Rest API to export financial reports in csv format"?

Reply:
By the way, the reports do not need to be financial at all.